sklearn/decomposition/tests/test_pca.py

```diff
@@ -532,6 +532,9 @@ def check_pca_float_dtype_preservation(svd_solver):
     assert pca_64.transform(X_64).dtype == np.float64
     assert pca_32.transform(X_32).dtype == np.float32
 
+    # the rtol is set such that the test passes on all platforms tested on
+    # conda-forge: PR#15775
```

Comment:
It takes me some time to figure out that this PR is in scikit-learn :)

Reply:
Yeah, but then conda-forge doesn't have 5 digit PRs, I think it's fine )